[15.0][ADD] ‎mail_partner_resolution_by_company (#944)
OCA/multi-company: mail_partner_company_resolve/readme/CONTEXT.rst

> 

Reviewer: ```suggestion
```

In models like project.task that interact directly with customers, incoming emails add 
CC addresses as followers, but the company restriction is not considered. In a multi-company
environment, if a customer sends an email with a CC address that belongs to a contact
dedicated to another company, that contact is added as a follower. Since the user lacks

> access rights to that contact’s company, they can no longer view the chatter.

Reviewer: ```suggestion
access rights to that contact’s company, they get an access error viewing the record.
```
